src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -1787,63 +1663,22 @@ resource pipeline_ExecuteETL 'Microsoft.DataFactory/factories/pipelines@2018-06-
 
 //------------------------------------------------------------------------------
 // msexports_ETL_ingestion pipeline
-// Trigger: msexports_ExecuteETL
+// Triggered by msexports_ExecuteETL
 //------------------------------------------------------------------------------
 @description('Transforms CSV data to a standard schema and converts to Parquet.')
 resource pipeline_ToIngestion 'Microsoft.DataFactory/factories/pipelines@2018-06-01' = {
   name: '${safeExportContainerName}_ETL_${safeIngestionContainerName}'
   parent: dataFactory
   properties: {
     activities: [
-      // (start) -> Wait -> FolderArray -> Scope -> Metric -> Date -> File -> Folder -> Delete Target -> Convert CSV -> Delete CSV -> (end)
-      // Wait
-      {
-        name: 'Wait'
-        type: 'Wait'
-        dependsOn: []
-        userProperties: []
-        typeProperties: {
-          waitTimeInSeconds: 60
-        }
-      }
-      // Set FolderArray
-      {
-        name: 'Set FolderArray'
-        type: 'SetVariable'
-        dependsOn: [
-          {
-            activity: 'Wait'
-            dependencyConditions: [
-              'Completed'
-            ]
-          }
-        ]
-        userProperties: []
-        typeProperties: {
-          variableName: 'folderArray'
-          value: {
-            value: '@split(pipeline().parameters.folderName, \'/\')'
-            type: 'Expression'
-          }
-        }
-      }
-      // Set Scope
```

Comment:
Can you bring these comments back? They make the file much easier to read and navigate.